services/tasks/local.go

```diff
@@ -478,14 +481,26 @@ func (l *local) Checkpoint(ctx context.Context, r *api.CheckpointTaskRequest, _
 	if err != nil {
 		return nil, err
 	}
-	image, err := ioutil.TempDir(os.Getenv("XDG_RUNTIME_DIR"), "ctd-checkpoint")
+	image, err := l.getCheckpointPath(container.Runtime.Name, r.Options)
 	if err != nil {
-		return nil, errdefs.ToGRPC(err)
+		return nil, err
+	}
+	jumpImage := true
```

Comment:
nit: perhaps `checkpointImageExists` might be more clear

Reply:
yes, have updated.